gta.go

```diff
@@ -89,6 +90,7 @@ type GTA struct {
 	packager Packager
 	prefixes []string
 	tags     []string
+	roots    []string
```

Comment:
The current behaviour is equivalent to the root being `/`, does it make sense to support _multiple_ roots?

Reply:
It does, because the traversal should stop at any entry in GOPATH, and GOPATH is a list of values.